drivers/hmis/spec/models/hmis/ce/match_candidate_pool_builder_spec.rb

```diff
@@ -3,10 +3,10 @@
 require 'rails_helper'
 
 RSpec.describe Hmis::Ce::Match::CandidatePoolBuilder do
-  let(:builder) { described_class.new }
-  let!(:opportunity) { create(:hmis_ce_opportunity) }
-  let(:project) { opportunity.project }
-  let(:organization) { project.organization }
+  let(:organization) { create :hmis_hud_organization }
+  let(:project) { create :hmis_hud_project, organization: organization }
+  let!(:opportunity) { create(:hmis_ce_opportunity, project: project) }
```

Comment:
is there a reason not to persist the organization and project using `let!` as opposed to `let`? I've seen this type of thing lead to some tricky-to-identify bugs.